### PR TITLE
NET-1534:upgrade Caddy to v2.8.4

### PR DESCRIPTION
**Pull request by yabinma**
## Checklist before requesting a review
- [ ] My changes affect only 10 files or less.
- [ ] I have performed a self-review of my code and tested it.
- [ ] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [ ] If it is a bugfix, my code is <= 200 lines.
- [ ] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [ ] Netmaker is awesome.

### CODE REVIEW

File: docker/Caddyfile [hunk 1]
```diff
@@ -25,6 +25,10 @@ https://api.{$NM_DOMAIN} {
...
 
 # MQ
-wss://broker.{$NM_DOMAIN} {
-	reverse_proxy ws://mq:8883 # For EMQX websockets use `reverse_proxy ws://mq:8083`
+broker.{$NM_DOMAIN} {
+	@ws {
+			header Connection *Upgrade*
+			header Upgrade websocket
+		}
+	reverse_proxy @ws mq:8883
```
Comment (abhishek9686) [inline docker/Caddyfile:33]: include comment for EMQX here

File: scripts/netmaker.default.env [hunk 2]
```diff
@@ -86,4 +86,8 @@ EMAIL_SENDER_ADDR=
 EMAIL_SENDER_USER=
 # sender smtp password
 EMAIL_SENDER_PASSWORD=
+# if batch peer update enable or not
+PEER_UPDATE_BATCH=true
+# batch peer update size when PEER_UPDATE_BATCH is enabled
+PEER_UPDATE_BATCH_SIZE=50
```
Comment (abhishek9686) [inline scripts/netmaker.default.env:92]: can you submit a separate PR for this to the v0.25.0 release branch?

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: docker/Caddyfile (hunk 1 above)
Comment (yabinma) [reply docker/Caddyfile]: done in new commit

File: scripts/netmaker.default.env (hunk 2 above)
Comment (yabinma) [reply scripts/netmaker.default.env:92]: yes, another PR https://github.com/gravitl/netmaker/pull/3101 to v0.25.0

**Review by abhishek9686 — APPROVED** [review]
